src/twister2/log_parser/harness_log_parser.py

```diff
@@ -18,7 +18,7 @@
 result_re_pattern: re.Pattern = re.compile(
     r'^.*(?P<result>PASS|FAIL|SKIP) - (test_)?(?P<testname>.*) in (?P<duration>[\d\.]+) seconds$'
 )
-testsuite_name_re_pattern: re.Pattern = re.compile(r'^.*Running test suite\s(?P<testsuite>.*)$')
+testsuite_name_re_pattern: re.Pattern = re.compile(r'^.*Running TESTSUITE\s(?P<testsuite>.*)$')
```

Comment:
should we use flag re.IGNORECASE for Pattern?

Reply:
I don't think so. This message was permanently changed from `test suite` into literally `TESTSUITE`, so I think that it can be more precisely to leave it without IGNORECASE flag.